t/16-markdown.t

```diff
@@ -3,12 +3,19 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 
 use Test::Most;
+use Test::Mojo;
 use Test::Warnings ':report_warnings';
 
 use FindBin;
 use lib "$FindBin::Bin/lib", "$FindBin::Bin/../external/os-autoinst-common/lib";
+use OpenQA::Test::Case;
 use OpenQA::Test::TimeLimit '10';
-use OpenQA::Markdown qw(bugref_to_markdown is_light_color markdown_to_html);
+use OpenQA::Markdown qw(bugref_to_html is_light_color markdown_to_html);
+
+my $test_case = OpenQA::Test::Case->new;
+my $schema = $test_case->init_data(fixtures_glob => '01-jobs.pl 03-users.pl 05-job_modules.pl');
```

Comment:
We did not need fixtures in this test in before. I am worried about the additional requirement and performance impact. What is the impact on runtime before and after this change?

Reply:
Removed that line as it was not needed.

Reply:
perfect :)